api-platform/admin-pack/1.0/post-install.txt

```diff
@@ -0,0 +1,16 @@
+<bg=blue;fg=white>                       </>
+<bg=blue;fg=white> To finish the install </>
+<bg=blue;fg=white>                       </>
+
+  * Your admin is almost ready:
+    1. Install the JavaScript dependencies by running <comment>yarn add @api-platform/admin</comment>
+    2. Go in <comment>webpack.config.js</comment> and <info>uncomment</info> the following lines:
+
+        Encore
+            // ...
+            .enableReactPreset()
+            .addEntry('admin', './assets/js/admin.js')
+
+  * Mark any entity with the <comment>@ApiResource</comment> annotation, it appears in the admin!
```

Comment:
Mark entities you want in the admin with the <comment>@ApiResource</comment> annotation